pallets/avn/src/tests/session_handler_tests.rs

```diff
@@ -1,167 +1,273 @@
 // Copyright 2022 Aventus Network Services (UK) Ltd.
 
 use crate::mock::*;
-use sp_avn_common::event_types::Validator;
 use sp_runtime::testing::UintAuthorityId;
 
-fn change_validators_good() {
-    VALIDATORS.with(|v| {
-        let mut v = v.borrow_mut();
-        *v = Some(vec![1, 2]);
-        Some(v.clone())
-    });
+fn avn_known_collators() -> sp_application_crypto::Vec<
```

Comment:
Do we need this function? can't we just use AVN::validators() where needed?
Alternatively can we use a type instead?

Reply:
Like
```
use AVN::validators as avn_known_collators;
```